src/python/nxt_python_asgi.h

```diff
@@ -22,7 +22,21 @@ typedef struct {
     nxt_unit_request_info_t  *req;
     uint64_t                 content_length;
 } nxt_py_asgi_add_field_ctx_t;
-
+#ifdef NXT_HAVE_ASYNCIO_RUNNER
```

Comment:
`#id defined()` is preferable over `#ifdef`.  However, since we can use just `#if ()`, it's better to use that (as in the other place).